git-config/src/values.rs

```diff
@@ -1183,63 +1097,84 @@ mod normalize {
 #[cfg(test)]
 mod boolean {
     use super::{Boolean, TrueVariant, TryFrom};
+    use nom::AsBytes;
 
     #[test]
     fn from_str_false() {
-        assert_eq!(Boolean::try_from("no"), Ok(Boolean::False("no".into())));
-        assert_eq!(Boolean::try_from("off"), Ok(Boolean::False("off".into())));
-        assert_eq!(Boolean::try_from("false"), Ok(Boolean::False("false".into())));
-        assert_eq!(Boolean::try_from("zero"), Ok(Boolean::False("zero".into())));
-        assert_eq!(Boolean::try_from("\"\""), Ok(Boolean::False("\"\"".into())));
+        assert_eq!(
+            Boolean::try_from("no".as_bytes()),
```

Comment:
Does `b"no"` not work here? If not, I have seen the `bstr` crate use `b("no")` with `b` being a tiny helper function.